### PR TITLE
Add tickets to spotlight card in homepage and profile page

### PR DESCRIPTION
**Pull request by vidya-ram**

### CODE REVIEW

File: funnel/templates/index.html.jinja2 [hunk 1]
```diff
@@ -91,7 +91,17 @@
...
+      {% if featured_project.features.tickets() -%}
+        tickets = {
+          boxofficeUrl: {{ config['BOXOFFICE_SERVER']|tojson }},
+          widgetElem: "#boxoffice-widget",
+          org: {{ featured_project.boxoffice_data.org|tojson }},
+          itemCollectionId: {{ featured_project.boxoffice_data.item_collection_id|tojson }},
+          itemCollectionTitle: {{ featured_project.title|tojson }}
+        };
+      {%- endif %}
+      window.Hasgeek.homeInit('.markdown', tickets);
```
Comment (jace) [inline funnel/templates/index.html.jinja2:104]: Do we need this when `tickets` is not defined? Couldn't these two lines go inside the if block?

**Review by jace — APPROVED** [review]
Looks okay, but one question.

File: funnel/templates/index.html.jinja2 (hunk 1 above)
Comment (vidya-ram) [reply funnel/templates/index.html.jinja2:104]: yes, `homeInit` has other fns required for homepage

File: funnel/templates/project_layout.html.jinja2 [hunk 2]
```diff
@@ -107,7 +107,7 @@
     <div class="top-padding bottom-padding project-details__box__title">
       <div class="project-banner__profile-details">
         {%- if project.profile.logo_url.url %}
-          <a href="{{ project.profile.url_for() }}" class="project-banner__profile-details__logo-wrapper" data-ga="View project">
+          <a href="{{ project.profile.url_for() }}" class="project-banner__profile-details__logo-wrapper" data-ga="View profile">
```
Comment (jace) [inline funnel/templates/project_layout.html.jinja2:110]: Change to "View account"

**Review by jace — DISMISSED** [review]

File: funnel/templates/profile_layout.html.jinja2 [hunk 3]
```diff
@@ -1,7 +1,257 @@
...
-{%- from "macros.html.jinja2" import img_size %}
+{%- from "macros.html.jinja2" import img_size, saveprojectform, calendarwidget, projectcard %}
 {% block title %}{{ profile.title }}{% endblock title %}
 
+{% macro featured_section(featured_project, heading=true) %}
+  {% if featured_project %}
+    <div class="bg-primary">
+    {% with current_sessions = featured_project.current_sessions() if
+    featured_project is not none else none %}
+    {% if current_sessions and current_sessions.sessions|length > 0 %}
```
Comment (jace) [inline funnel/templates/profile_layout.html.jinja2:10]: If it's a list the second clause isn't needed.

**Pull request merged**